spec/app/domain/authentication/authn_k8s/validate_pod_request_spec.rb

```diff
@@ -42,13 +42,13 @@
                              :spiffe_id              => spiffe_id) }
   let(:pod_spec) { double("PodSpec") }
   let(:pod) { double("Pod", :spec => pod_spec) }
-  
+
   let(:k8s_object_lookup_class) { double("K8sObjectLookup") }
 
   let(:validate_application_identity) { double("ValidateApplicationIdentity") }
 
-  let(:dependencies) { { resource_class:             double(),
-                         k8s_object_lookup_class:   k8s_object_lookup_class,
+  let(:dependencies) { { resource_class:                double(),
```

Comment:
Formatting nits:

For multiline statements ruby style guide prescribes `do.. end` over `{...}`.

Separately, I'd opt for the opening / closing braces of hash on separate lines, though I'm unsure what the guidelines are on that, just stating my taste on that one...
```
let(:dependencies) do 
  { 
    resource_class:                double(),
    k8s_object_lookup_class:       k8s_object_lookup_class,
    validate_application_identity: validate_application_identity
  }
end
```
I really want to automate all this formatting stuff... it's a time waster.  I have a possible solution I'll be posting about today.